assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/ModalContainer.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { sprintf, __ } from '@wordpress/i18n';
+
+import { TicketAssignmentsManagerModal } from '../components';
+import { ModalContainerProps } from '../types';
+import { withContext } from '../context';
+
+const ModalContainer: React.FC<ModalContainerProps> = ({ isOpen, onClose, ...props }) => {
+	const { assignmentType, entity } = props;
+
+	let title = '';
+	if (assignmentType === 'forDate') {
+		title = sprintf(__('Ticket Assignment Manager for Datetime: %s - %s'), entity.dbId, entity.name);
+	} else if (assignmentType === 'forTicket') {
+		title = sprintf(__('Ticket Assignment Manager for Ticket: %s - %s'), entity.dbId, entity.name);
+	}
+
+	return isOpen && withContext(TicketAssignmentsManagerModal, { ...props, title, onCloseModal: onClose });
```

Comment:
not sure which context is picking `withContext` here

Reply:
TAM context